src/pricecypher/contracts/QualityTestScript.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+from typing import Optional, Any
+
+from pricecypher.contracts import Script, TestSuite
+
+
+class QualityTestScript(Script, ABC):
+    """
+    The abstract QualityTestScript class serves as an interaction contract such that by extending it with its
+        methods implemented, a script can be created that performs data quality tests on a dataset, which can then be
+        used in a generalized yet controlled setting.
+    """
+
+    def execute(self, business_cell_id: Optional[int], bearer_token: str, user_input: dict[Any: Any]) -> Any:
+        return self.execute_tests(business_cell_id, bearer_token)
```

Comment:
I have no clue whether they exist. But it really doesn't matter because it cannot be an interface since it needs to be an abstract class to implement its parent `execute` function.